### ISSUE TITLE
[bug] Python integration On Sequence receives `"params": { "command": "send_cmd" }`

### ISSUE THREAD

**Issue by ijc** [unfoldedcircle/bug-reports#381]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

I have just published a Python [integration for MythTV](https://github.com/ijc/uc-integration-mythtv).

It mostly works well but when I try and use a simple command on an On/Off sequence instead of getting a message with the desired command I get:

```json
{
  "kind": "req",
  "id": 5,
  "msg": "entity_command",
  "msg_data": {
    "cmd_id": "send_cmd",
    "entity_id": "mythtv",
    "entity_type": "remote",
    "params": {
      "command": "send_cmd"
    }
  }
}
```
 
Notice the `"command": "send_cmd"` which appears instead of whatever command was supposed to appear.

When a command is sent by a button (hard or soft) in the UI then it works.

### How to Reproduce

- Install the integration and point it at a MythTV system (or hack `mytvtv.py` to fake one I suppose).
- Configure an activity with a On Sequence using a command from the integration
- Switch to that activity on the remote

### Expected behavior

I expect the message received by the integration to contain the right command name somewhere, if not the "command" field then somewhere else.

### System version

UI 0.42.1, core 0.43.0-beta, system 1.7.12

### What part of the system affected by the problem?

Integration

**Comment by zehnm** [comment]
I'll check this tomorrow

**Comment by ijc** [comment]
Thanks! I did have a dig around in https://github.com/unfoldedcircle/remote-ui but that only appears to have v0.38.4 compared with 0.42.1 on the remote and in any case I failed to spot anything relevant.

**Comment by zehnm** [comment]
I'll check remote-ui as well, that should be up-to-date, otherwise we'll update it asap.

**Comment by zehnm** [comment]
Confirmed bug, I could reproduce it with the provided remote.py example in the Python integration driver.

- `Send command` actions work
- Simple commands provided by the integration are sent in a wrong format as reported in this issue.

As a workaround until a bugfix is released, the simple commands can be wrapped in a `Send command`:

<img width="476" alt="image" src="https://github.com/unfoldedcircle/feature-and-bug-tracker/assets/17356629/a8bc14fb-7ce2-47be-be40-5629c0698fa1">

- First `HOME` action doesn't work
- Second action works if the `HOME` command is wrapped into `Send command`

**Comment by ijc** [comment]
Awesome, thanks for the workaround!

**Comment by zehnm** [comment]
Fixed in next release. A beta bugfix release is scheduled for tomorrow, a public release soon afterwards.

**Comment by zehnm** [comment]
Fixed in beta 1.7.13 and I'll close this issue once a public release is out.

**Comment by zehnm** [comment]
Part of 1.7.14
